Introduce Kafka alerts tests

francoisferrand opened the pull request:
Also fix minor issue in ConsumerLag warning description.

Issue: ZENKO-3941

francoisferrand committed:
Introduce Kafka alerts tests

Also fix minor issue in ConsumerLag warning description.

Issue: ZENKO-3941

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/zenko/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `feature/ZENKO-3941-test-kafka-alerts` into
`development/2.2`
* [w/2.3/feature/ZENKO-3941-test-kafka-alerts](https://github.com/scality/zenko/compare/development/2.3...w/2.3/feature/ZENKO-3941-test-kafka-alerts) will be merged into `development/2.3` (pull request #1508)



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

Commit:
ZENKO-3941 - update alert to use `absent_over_time`

When no more values are exposed for a metric, this metric stop
existing. In this case you must use `absent` or `absent_over_time`
to trigger on the missing metric.

bertranddemiddelaer commented:
/bypass_author_approval

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/495a559e1275a16e4013bec26d37d5cf47e0b3e2) did not succeed in branch w/2.3/feature/ZENKO-3941-test-kafka-alerts.




*The following options are set:* **bypass_author_approval**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **bypass_author_approval**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.2`

* :heavy_check_mark: `development/2.3`



The following branches have **NOT** changed:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/2.1`



Please check the status of the associated issue ZENKO-3941.

Goodbye francoisferrand.
